styles/wikiwand/catppuccin.user.css

```
@@ -368,6 +398,9 @@
     a.wl {
       color: @accent-color !important;
     }
+    pre, code {
```

```suggestion
    pre, code, samp, kbd {
```
From your example above actually.